client/my-sites/plugins/marketplace/marketplace-domain-upsell/styles.scss

```diff
@@ -0,0 +1,104 @@
+//@import '~@automattic/marketplace/styles/mixins';
+
+.marketplace-domain-upsell__root {
+	margin-left: 34px;
+	display: grid;
+	grid-template-rows: 40px 100%;
+	grid-template-columns: 60% 40%;
+
+	*,
+	*::before,
+	*::after {
+		box-sizing: border-box;
+	}
+	
+	.marketplace-domain-upsell__shopping-cart-container {
+		padding-left: 50px;
+		max-width: 400px;
+	}
+
+	.marketplace-domain-upsell__header {
+		margin: 0 0 20px;
+		@media ( min-width: 782px ) {
+			margin: 0 0 20px;
+		}
+		@media ( min-width: 600px ) {
+			margin: 0 0 20px;
+		}
+	}
+	.marketplace-title {
+		font-family: var( --p2-font-inter );
```

Comment:
Why does the page use this custom font? Is there a reason why to not use Calypso defaults?

Reply:
Fixed by 0aa0786